### PR TITLE
React router

### PR DESCRIPTION
**Pull request by flagoon**
Hey, @binygal I remade my app to use react-router instead of redux. It's cleaner for now. Refactor was connected with removing most of the tests, as they were testing nothing, but I added tests for react-router. After this merge, the next step is component with views with mockedBooks. I will start doing it Wednesday, as tomorrow I have a busy day.

ps: sorry for styling changes, but I think I have my vscode under control now.

Cheers!

### CODE REVIEW

File: src/Components/App.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import React from "react";
+import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
```
Comment (binygal) [inline src/Components/App.tsx]: Why do you add this import of you're not using the imported objects?

File: src/Components/MainArea/MainArea.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import Bookshelf from './Bookshelf/Bookshelf';
+
+const MainArea = () => {
+    return (
+        <Switch>
+            <Route exact path='/' component={Bookshelf} />
+            <Route path='/bookshelf' component={Bookshelf} />
+            <Route path='/archive' component={Archive} />
+            <Route path='/addbook' component={AddBookForm} />
+            <Route component={NoMatch} />
```
Comment (binygal) [inline src/Components/MainArea/MainArea.tsx:15]: gg It's important to develop the not so optimistic approach 😄

File: src/Components/Navbar/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    </Link>
+    <Link to="/addbook" style={{ textDecoration: "none" }}>
+      <HeaderLink>Add book</HeaderLink>
+    </Link>
+  </StyledHeader>
+);
+
+export default Navbar;
+
+const StyledHeader = styled.ul`
```
Comment (binygal) [inline src/Components/Navbar/Navbar.tsx]: For code readability I always prefer to declare any function/object that I'm using before actually using it.

**Review by binygal — APPROVED** [review]
Looks good! Please see the notes.

File: src/Components/App.tsx (hunk 1 above)
Comment (flagoon) [reply src/Components/App.tsx]: I must have missed that. I create tslint rule for unused variables plus I added husky to check this kind of thing before commit (also tests).

File: src/Components/Navbar/Navbar.tsx (hunk 3 above)
Comment (flagoon) [reply src/Components/Navbar/Navbar.tsx]: I moved all styled components to another file, called <componentName>.styled.js and import it.

**Pull request merged**